code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -287,14 +270,6 @@
 		tracks[initial(track.name)] = track
 	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(tracks))
 
-/datum/gear/suit/puffer_coat
-	display_name = "puffer coat"
-	path = /obj/item/clothing/suit/jacket/puffer
-
-/datum/gear/suit/puffer_vest
```

Comment:
I know of people who use these, did you provide a suitable replacement via geartweaks or is this just another undocumented change? This isn't redundant, so the change is undocumented.